### PR TITLE
Dev 47

### PR DESCRIPTION
**Pull request by ajkarczewski**
### What (if any) features are you implementing?
- Created UI and route for the Industry Selection component

### What (if anything) did you refactor?
- N/A
### Were there any issues that arose?
- N/A

### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?

### CODE REVIEW

**Review by YashsviG — CHANGES_REQUESTED** [review]
Please add get and patch requests from the BE to have the data saved in the db

File: frontend/src/App.js [hunk 1]
```diff
@@ -29,6 +30,8 @@ const App = () => {
           <Route path="/setup-profile-2" element={<SecondProgressBarForm />} />
           <Route path="/setup-profile-3" element={<ThirdProgressBarForm />} />
           <Route path="/mentee-flow-1" element={<MenteeRole />} />
+          <Route path="/industry-selection" element={ <IndustrySelection matchedWith="mentee" industryQuestion="In which industries do you have experience? Select up to 5!" />} />
+          <Route path="/industry-selection" element={<IndustrySelection matchedWith="mentor" industryQuestion="Select up to 5 industries of interest!" />} />
```
Comment (YashsviG) [inline frontend/src/App.js:34]: follow the same pathing we are doing for the rest of the components

File: frontend/src/Components/RoleSelection.js [hunk 2]
```diff
@@ -92,11 +93,11 @@ const RoleSelection = ({ question, matchedWith }) => {
...
-      //  if (title === "Mentor") {
-      navigate("/mentor-flow-4");
-      //  } else if (role === "Mentee") {
-      //    navigate("/mentee-flow-1");
-      //  }
+       if (title === "Mentor") {
+      navigate("/mentor-flow-2");
+       } else if (title === "Mentee") {
+         navigate("/mentee-flow-2");
+       }
```
Comment (YashsviG) [inline frontend/src/Components/RoleSelection.js:100]: we can just do else block like:
```suggestion
       } else {
         navigate("/mentee-flow-2");
       }
```

File: frontend/src/Components/IndustrySelection.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        maxBodyLength: Infinity,
+        url: `${api.users.userProfile}/${userId}`,
+        headers: {
+          Authorization: `Bearer ${token}`,
+          "Content-Type": "application/json",
+        },
+        data: data,
+      };
+      const response = await axios.request(config);
+      console.log(response.data);
```
Comment (YashsviG) [inline frontend/src/Components/IndustrySelection.js:98]: TODO: Need to add a navigate here for the upcoming flow-3
navigate('mentor/mentee-flow-3')

File: frontend/src/App.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ import {
   Calendly
 } from "./Components";
 import MatchFlow from "./constants/MatchFlow";
+import IndustrySelection from "./Components/IndustrySelection";
```
[the remark this answers is not in the document]
Comment (ajkarczewski) [reply frontend/src/App.js:15]: can you clarify what you mean in the first part?

File: frontend/src/App.js [hunk 5]
```diff
@@ -42,12 +43,28 @@ const App = () => {
...
+        }
+        />
+         <Route path="/mentee-flow-2" 
+         element={
+         <IndustrySelection
+          matchedWith="mentor"
+          question={MatchFlow.menteeQues2} 
+          />
+        } 
+        />
```
Comment (YashsviG) [inline frontend/src/App.js:67]: Love the clean up you have done in this file :))

File: frontend/src/Components/RoleSelection.js [hunk 6]
```diff
@@ -33,12 +35,13 @@ const useStyles = makeStyles({
   },
 });
 
-const RoleSelection = ({ question, matchedWith }) => {
+const TitleSelection = ({ question, matchedWith }) => {
```
Comment (YashsviG) [inline frontend/src/Components/RoleSelection.js:38]: Change the name of the file to TitleSelection as well

**Review by YashsviG — COMMENTED** [review]
Some small stuff to fix other than that looks good :))
Do you have the CSS all in place? or still working on it?

**Review by YashsviG — APPROVED** [review]

**Pull request merged**